plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/compare/changedList/ChangedFileNode.java

```diff
@@ -27,24 +28,33 @@
  *
  * @author Igor Vinokur
  */
-public class ChangedNode extends AbstractTreeNode implements HasPresentation, HasAction {
-
-    private String name;
-    private String state;
+public class ChangedFileNode extends AbstractTreeNode implements HasPresentation, HasAction {
 
+    private String           pathName;
+    private String           state;
```

Comment:
May be it would be better to make state as `enum` or specialized constant? 
For example how it done in another IDE [FileStatus.java](https://github.com/JetBrains/intellij-community/blob/master/platform/editor-ui-api/src/com/intellij/openapi/vcs/FileStatus.java)